jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardWithHeaderAndFooterTests.java

```diff
@@ -0,0 +1,140 @@
+    private String textFooterCardWithHeaderAndFooter = "2 days ago";
+
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void baseValidationCardWithHeaderTest() {
+        baseValidation(cardWithHeader);
+    }
+
+    @Test
+    public void baseValidationCardWithHeaderAndFooterQuoteTest() {
+        baseValidation(cardWithHeaderAndFooterQuote);
+    }
+
+    @Test
+    public void baseValidationCardWithHeaderAndFooterTest() {
+        baseValidation(cardWithHeaderAndFooter);
+    }
+
+
+    //test title
+    @Test
+    public void getTitleTextCardWithHeaderTest() {
+        cardWithHeader.title.is().text(textTitleCardWithHeader);
+    }
+
+
+    @Test
+    public void getTitleTextCardWithHeaderAndFooterTest() {
+        cardWithHeaderAndFooter.title.is().text(textTitleCardWithHeaderAndFooter);
+    }
+
+    //test paragraph
+    @Test
+    public void getParagraphTextCardWithHeaderTest() {
+        cardWithHeader.paragraph.is().text(textParagraphCardWithHeader);
+    }
+
+    @Test
+    public void getParagraphTextCardWithHeaderAndFooterQuoteTest() {
+        cardWithHeaderAndFooterQuote.paragraph.is().text(textParagraphCardWithHeaderAndFooterQuote);
+    }
+
+    @Test
+    public void getParagraphTextCardWithHeaderAndFooterTest() {
+        cardWithHeaderAndFooter.paragraph.is().text(textParagraphCardWithHeaderAndFooter);
+    }
+
+    //test button
+    @Test
+    public void getButtonTextCardWithHeaderTest() {
+        cardWithHeader.button.is().text(textButtonCardWithHeader);
+    }
+
+    @Test
+    public void getButtonTextCardWithHeaderAndFooterTest() {
+        cardWithHeaderAndFooter.button.is().text(textButtonCardWithHeaderAndFooter);
+    }
+
+//test header
+    @Test
+    public void getHeaderTextCardWithHeaderTest() {
+        cardWithHeader.header.is().text(textHeaderCardWithHeader);
+    }
+
+    @Test
+    public void getHeaderTextCardWithHeaderAndFooterQuoteTest() {
+        cardWithHeaderAndFooterQuote.header.is().text(textHeaderCardWithHeaderAndFooterQuote);
+    }
+
+    @Test
+    public void getHeaderTextCardWithHeaderAndFooterTest() {
+        cardWithHeaderAndFooter.header.is().text(textHeaderCardWithHeaderAndFooter);
+    }
+
+//test footer
+    @Test
+    public void getFooterTextCardWithHeaderTest() {
+        cardWithHeader.footer.is().notAppear();
+    }
+
+    @Test
+    public void getFooterTextCardWithHeaderAndFooterQuoteTest() {
+        cardWithHeaderAndFooterQuote.footer.is().text(textFooterCardWithHeaderAndFooterQuote);
+    }
+
+    @Test
+    public void getFooterTextCardWithHeaderAndFooterTest() {
+        cardWithHeaderAndFooter.footer.is().text(textFooterCardWithHeaderAndFooter);
+    }
+
+    @Test
+    public void clickButtonCardWithHeaderTest() {
+        cardWithHeader.button.click();
+        validateAlert(is("Button Clicked!"));
+        cardWithHeader.button.isEnabled();
```

Comment:
it's not a check. this method can return and true and false.
to check that element is enabled and is displayed we should use method is(), for example:
```
       cardWithHeader
                .button
                .is()
                .enabled()
                .displayed();
```